<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>category_widget</class>
 <widget class="QWidget" name="category_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Category</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTreeWidget" name="tree_category">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>401</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>401</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Добавить новую категорию</string>
      </property>
      <widget class="QLabel" name="lbl_name">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>180</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Имя категории</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_parent">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>180</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Родительская категория</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="led_name">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>180</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cmb_category">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>60</y>
         <width>180</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_add_category">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить категорию</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
